app/serializers/holiday_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class HolidaySerializer < ActiveModel::Serializer
+  attribute :title
+  attribute :start do
```

Comment:
You can use the `key` option for aliases instead and avoid yielding a block here and for `end` too.